interfaces/builtin/netplan_observe.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const netplanObserveConnectedPlugAppArmor = `
+# Description: Can read netplan configuration files
+# Usage: reserved
+
+/etc/netplan r,
```

Comment:
I think changing the name out of netplan-observe will probably blur the lines of what this interface is supposed to be doing.  "network configuration" is surprisingly hard to tangibly define So somebody will come in, ask for more to be there, and we'll be in trouble because we don't know whether we should or not hand it off. I'd prefer to keep the clear scope on this one, and if we need something more generic we can work through it.